javascript/packages/orchestrator/src/orchestrator.ts

```diff
@@ -344,211 +349,13 @@ export async function start(
-          remoteFilePath: `${client.remoteDir}/${override.remote_name}`,
-        });
-      }
-
-      let keystoreLocalDir;
-      if (node.accounts) {
-        // check if the node directory exists if not create (e.g for k8s provider)
-        let nodeFilesPath = tmpDir.path;
-        if (parachain && parachain.name) nodeFilesPath += `/${parachain.name}`;
-        nodeFilesPath += `/${node.name}`;
-
-        await makeDir(nodeFilesPath, true);
-
-        const isStatemint = parachain && parachain.chain?.includes("statemint");
-        const keystoreFiles = await generateKeystoreFiles(
-          node,
-          nodeFilesPath,
-          isStatemint,
-        );
-        keystoreLocalDir = path.dirname(keystoreFiles[0]);
-      }
-
-      // replace all network references in command
-      replaceNetworkRef(podDef, network);
-
-      await client.spawnFromDef(
-        podDef,
-        finalFilesToCopyToNode,
-        keystoreLocalDir,
-        parachainSpecId || client.chainId,
-        node.dbSnapshot,
-      );
-
-      const [nodeIp, nodePort] = await client.getNodeInfo(podDef.metadata.name);
-      const nodeMultiAddress = await generateBootnodeString(
-        node.key!,
-        node.args,
-        nodeIp,
-        nodePort,
-        true,
-        node.p2pCertHash,
-      );
-      multiAddressByNode[podDef.metadata.name] = nodeMultiAddress;
-
-      if (node.addToBootnodes) {
-        bootnodes.push(nodeMultiAddress);
-        await addBootNodes(chainSpecFullPath, bootnodes);
-      }
-
-      let networkNode: NetworkNode;
-
-      const endpointPort = RPC_WS_PORT;
-      if (options?.inCI) {
-        const nodeIp = await client.getNodeIP(podDef.metadata.name);
-        networkNode = new NetworkNode(
-          node.name,
-          WS_URI_PATTERN.replace("{{IP}}", nodeIp).replace(
-            "{{PORT}}",
-            endpointPort.toString(),
-          ),
-          METRICS_URI_PATTERN.replace("{{IP}}", nodeIp).replace(
-            "{{PORT}}",
-            PROMETHEUS_PORT.toString(),
-          ),
-          nodeMultiAddress,
-          userDefinedTypes,
-        );
-      } else {
-        const nodeIdentifier = `${podDef.kind}/${podDef.metadata.name}`;
-        const fwdPort = await client.startPortForwarding(
-          endpointPort,
-          nodeIdentifier,
-        );
-        const nodePrometheusPort = await client.startPortForwarding(
-          PROMETHEUS_PORT,
-          nodeIdentifier,
-        );
-
-        const listeningIp = networkSpec.settings.local_ip || LOCALHOST;
-
-        networkNode = new NetworkNode(
-          node.name,
-          WS_URI_PATTERN.replace("{{IP}}", listeningIp).replace(
-            "{{PORT}}",
-            fwdPort.toString(),
-          ),
-          METRICS_URI_PATTERN.replace("{{IP}}", listeningIp).replace(
-            "{{PORT}}",
-            nodePrometheusPort.toString(),
-          ),
-          nodeMultiAddress,
-          userDefinedTypes,
```

Comment:
Where all this code went ?

Reply:
🤔 Isnt this the code that @pepoviola moved to `spawner` through the PR #879 ?

Reply:
yes, is part of the spawner now.